pkg/database/database.go

```diff
@@ -394,13 +394,32 @@ func checkAndMigrateDB(db *DbInstance, migrate bool) error {
 				}); err != nil {
 					return fmt.Errorf("error migrating database to FastTrackML schema 1ce8669664d2: %w", err)
 				}
+				fallthrough
+
+			case "1ce8669664d2":
```

Comment:
just curious -> how we select migration id?  `5d042539be4f` probably randomly but ....?

Reply:
Yep, randomly